gtsam/sfm/MFAS.cpp

```diff
@@ -111,10 +113,8 @@ void removeNodeFromGraph(const Key node,
   graph.erase(node);
 }
 
-MFAS::MFAS(const std::shared_ptr<vector<Key>>& nodes,
-           const TranslationEdges& relativeTranslations,
-           const Unit3& projectionDirection)
-    : nodes_(nodes) {
+MFAS::MFAS(const TranslationEdges& relativeTranslations,
+           const Unit3& projectionDirection) {
```

Comment:
are the `nodes` no longer required here? are they initialized elsewhere?

Reply:
Nodes are no longer required. They were not being used since I refactored this class a while ago. I had missed removing this unused member then.